domain/src/main/java/com/github/leuludyha/domain/model/user/MainUser.kt

```diff
@@ -6,10 +6,10 @@ import com.github.leuludyha.domain.model.user.preferences.WorkPreference
 import kotlinx.coroutines.flow.Flow
 
 data class MainUser(
-    val userId: String,
+    var userId: String,
     override var username: String,
-    override val profilePictureUrl: String,
-    override val phoneNumber: String?,
+    override var profilePictureUrl: String,
+    override var phoneNumber: String?,
```

Comment:
You didn't find any alternative to using `var`?

Reply:
Unfortunately the values may change because the user is initialized as soon as the authentication user is used for the first time (because of injection), but the first time the injection is needed the user may not have network connection, may not have stored values in persistent memory or a combination of those. If you have an idea, I am open to it, since I am struggling to find something clean.